### PR TITLE
windows support for interactive mode

### PR DESCRIPTION
**Pull request by James-Pickett**
updates interactive mode to use named pipes for windows

### CODE REVIEW

File: pkg/osquery/interactive/interactive_test.go [hunk 1]
```diff
@@ -1,3 +1,4 @@
+// disabled on windows because go tests doesn't get access to used named pipes
```
Comment (directionless) [inline pkg/osquery/interactive/interactive_test.go:1]: Really!?

**Review by directionless — COMMENTED** [review]
Code looks fine. I assume CI is correct

File: pkg/osquery/interactive/interactive_test.go (hunk 1 above)
Comment (James-Pickett) [reply pkg/osquery/interactive/interactive_test.go]: @directionless , I looked into this deeper and this turned out to be an issue with the tests trying to download the osqueryd binary in parallel to the same location. I moved the initial download and caching of the osqueryd binary to the TestMain func and it cleared up the issue.

**Review by directionless — APPROVED** [review]

File: pkg/osquery/interactive/interactive_test.go (hunk 1 above)
Comment (James-Pickett) [reply pkg/osquery/interactive/interactive_test.go]: @directionless , nevermind, I forgot to call m.Run() at the end of test main .... so the tests didnt run ... back to the drawing board.

**Review by directionless — DISMISSED** [review]
Seems fine. Sometime we should solve the windows test issue, but it can be not-today

**Review by directionless — APPROVED** [review]

**Pull request merged**